tests/test_data.py

```diff
@@ -0,0 +1,118 @@
+from conftest import skipif_yask
```

Comment:
Note for reviewers: some of these tests were moved from ``test_yask.py``, and are now also executed by the ``core`` backend